### PR TITLE
Map guide

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for the issue #999

### CODE REVIEW

File: docs/src/user/guide/map.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+![map page](./img/map-2.png)
+
+## How can I hide/unhide the legend?
+
+By default, the legend is shown on the map page but you can hide the legend for a cleaner view by clicking on the 1️⃣ `Arrow` icon.
+
+![legend](./img/map-3.png)
+
+Upon clicking, the legend will be hidden.
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/map.md]:
```suggestion
Upon clicking the icon, the legend will be hidden.
```

File: docs/src/user/guide/map.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+![legend](./img/map-4.png)
+
+## How can I search for the site?
+
+You can search for the desired site on the map by typing into the 1️⃣ search bar located at the top right corner.
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/map.md]:
```suggestion
You can search for the desired site on the map by typing into the 1️⃣ search bar located in the top right corner.
```

File: docs/src/user/guide/map.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+By default, the miniSASS Observation (Overlay layer) and the Default (Base layer) is selected to show the map but you can change the layer for a different view.
+
+![default layer](./img/map-8.png)
+
+To change the layer click on the 1️⃣ layer icon available in the bottom left corner.
+
+![layer icon](./img/map-9.png)
+
+Upon clicking the popup will open, where you can select the layer. To select the `Satellite` layer click on the 1️⃣ radio button available in front of the satellite.
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/map.md]:
```suggestion
Upon clicking the icon, the popup will open, where you can select the layer. To select the `Satellite` layer click on the 1️⃣ radio button available in front of the satellite.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]
Spelling was good, just some minor language adjustments.

**Review by Jeremy-Prior — APPROVED** [review]

**Comment by Jeremy-Prior** [comment]
@ketankartoza Please address the conflicts when possible

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 38 sec

**Pull request merged**